erpnext/projects/web_form/tasks/tasks.py

```diff
@@ -3,10 +3,10 @@
 import frappe
 
 def get_context(context):
-	if frappe.form_dict.project:
-		context.parents = [{'title': frappe.form_dict.project, 'route': '/projects?project='+ frappe.form_dict.project}]
-		context.success_url = "/projects?project=" + frappe.form_dict.project
+	if frappe.form_dict.default_project:
+		context.parents = [{'title': frappe.form_dict.default_project, 'route': '/projects?project='+ frappe.form_dict.default_project}]
+		context.success_url = "/projects?project=" + frappe.form_dict.default_project
 		
-	elif context.doc and context.doc.get('project'):
-		context.parents = [{'title': context.doc.project, 'route': '/projects?project='+ context.doc.project}]
-		context.success_url = "/projects?project=" + context.doc.project
+	elif context.doc and context.doc.get('default_project'):
```

Comment:
Under **context.doc.get('default_project')** we are getting correct project name
![context_project](https://user-images.githubusercontent.com/80836439/132480790-a291b3ec-f1d3-4080-8df5-62f963f24f8b.PNG)